.github/workflows/ci.yml

```diff
@@ -85,9 +90,13 @@ jobs:
         with:
           path: ${{ steps.yarn-cache.outputs.dir }}
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+      - name: Get arch value
+        run: echo "RCT_NEW_ARCH_ENABLED=${{ matrix.arch == 'NewArch' && '1' || '0' }}" >> $GITHUB_ENV
       - name: Install Dependencies
         run: yarn && yarn bootstrap
       - name: Install Podfiles
         run: cd example && npx pod-install
+      - name: Print React Native Info
+        run: cd example && npx react-native info
       - name: Build example app
-        run: cd example && yarn ios
+        run: cd example && yarn ios --no-packager
```

Comment:
🍒 With `--no-packager` we skip launching local server, which originally tried to launch it in separate terminal window. this led to the error `Cannot start server in new window because no terminal app was specified.
`